Web3privacynowplatform/Briefs/Waku.md

```diff
@@ -0,0 +1,88 @@
+**Basic info**
+
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Project name**  | Waku |
+| **Logotype**  | https://github.com/waku-org/docs.waku.org/blob/develop/static/img/logo_white.svg |
+| **Short project description (1 sentence)** | A family of robust, censorship-resistant communication protocols designed to enable privacy-focused messaging for web3 apps. |
+| **Launch date** | March 2022  |
+| **Is it open source?** | Yes |
+| **Website**  | https://waku.org |
+| **Github**  | https://github.com/waku-org |
+
+**Token**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Do you have a token** | No |
+
+**Socials**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Blog** | https://blog.waku.org/ |
+| **Twitter (X)** | https://x.com/waku_org/waku_org |
+| **Telegram** | https://t.me/waku_org |
+| **Discord** | https://discord.waku.org |
+| **Other** | https://logos.co |
+
+**Docs**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Docs** | https://docs.waku.org |
+
+**Team**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Is your team anon or public**  | public | 
+| **List core contributors** | https://github.com/orgs/waku-org/people |
+
+**Funding** -- NEEDS REVIEW
+| Request  | Data point | 
+| ------------- | ------------- |
+| **How are you funded**  | mixed | 
+| **List VCs backing you**  | Logos Collective |
+| **List VCs round**  | if aplicable: pre-seed, seed, Series A, B,C | 
+| **Raised money** | add a Crunchbase link | 
+
+**Github** -- NEEDS REVIEW
+| Request  | Data point | 
+| ------------- | ------------- |
+| **What licence is in use**  | add link to Github repo | 
+
+**Technology** -- NEEDS REVIEW
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Technical specialisation**  | specify: ZK, account abstraction, mixnet etc | 
+| **Specify technology readiness**  | infra: mainnet, apps: alpha | 
+| **Features**  | peer-to-peer, decentralized, censorship-resistance | 
+| **Encryption method**  | !!! | 
+
+**Privacy** -- NEEDS REVIEW
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Is it peer-to-peer**  | yes | 
+| **Complience**  | no |
+| **KYC**  | no |
+| **Privacy policy** | https://waku.org/terms |
+| **Collected data**  | specify what data do you collect |
+| **Data sharing** | specify what data you share with third-parties |
+| **Sign-in requirments** | specify what data is needed to start using service |
+| **Identity integrations** | do you use third-party identity services (like ENS, Gitcoin Passport etc) |
+
+**Security** -- NEEDS REVIEW
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Asset custody** | specify | 
+| **Upgradability**  | specify |
+| **Social dependency**  | team, CEO |
+| **Third-parties dependency** | specify |
+| **Technical dependency** | specify: Ethereum, ZK etc |
+| **Have you been audited by a third-party** | yes/no |
```

Comment:
RLN-v2 has been audited, btw :)

Reply:
For the scoring process we can not consider nuanced cases, so best to put 'no' unless the whole code-base has been audted at this time.

This can always be updated in future, if a user is interested in Waku they will then click on links and engage in DYOR